src/zenml/model/base_model.py

```diff
@@ -0,0 +1,61 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Model base model to support Model WatchTower feature."""
+
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+
+STR_FIELD_MAX_LENGTH = 255
+TEXT_FIELD_MAX_LENGTH = 65535
```

Comment:
We already have both of these constants in `/Users/strickvl/coding/zenml/repos/zenml/src/zenml/models/constants.py` FYI

Reply:
Yeah, but I had to move `ModelBaseModel` out of `models` to break circular loops, thus redefined these here. Not the best one, but serves the purpose.

Reply:
Strong opinion that this should be kept in `models/` and I'm sure we can find a cleaner way to break the circular imports (send me the stack trace if you need help on this)

Reply:
This is my way to approach this: https://github.com/zenml-io/zenml/pull/1819/commits/0fa0d5a78bf77376b28e945e55a234ccf0bccaf5
In short: avoid exposing it out of the modules and use by full link from the file aside. Looks rather good to me.